### PR TITLE
Improve README with description of project

### PR DESCRIPTION
**Pull request by BradLewis**
PR for updating the README with more description about the project. Leaving it as a draft atm as I imagine we'll have a lot of discussions for this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,11 +19,25 @@ A <em>blazingly fast</em> web application for generating LastFM collages, writte
...
+
+- **Dimensions**: specify the exact number of rows and columns you would like within your collage.
+- **Text**: Choose between adding the album name, artist name and playcount to your collage; or any combo you choose.
+- **Image Size**: Know the dimensions of the collage you need? SongStitch allows you to specify the desired collage image dimensions to ensure you only get what you need. You can even enable lossy compression!
+
+Have a suggestion on how we can make SongStitch better? Feel free to create an issue on [GitHub](https://github.com/SongStitch/song-stitch/issues/new), or submit a PR!
+
 ## Usage
 
+Below are the instructions on how you can run SongStitch yourself. You can either run it with `go` directly (we're currently using `v1.20.4`), or you can run it with `docker` and `docker-compose`.
```
Comment (TheDen) [inline README.md:39]: Maybe we shouldn't specify the exact version? It'll be a lot of busywork to always update it in the docs

Comment (BradLewis) [reply README.md:39]: Yeah can specify just 1.20? This is less of a requirement and more of just a "this is the version we use"

Comment (TheDen) [reply README.md:39]: What if we just add this bade?

GitHub go.mod Go version

Comment (BradLewis) [reply README.md:39]: Yeah that'll work

**Review by TheDen — APPROVED** [review]

**Pull request merged**